contents/codes/305.md

```diff
@@ -3,6 +3,7 @@ set: 3
 code: 305
 title: Use Proxy
 #references:
+#   "Rust HTTP Status Constant": "http::StatusCode::USE_PROXY"
```

Comment:
I wonder why the references in this file were commented out. I'll look into that.